samigo/samigo-app/src/webapp/include/header.inc

```diff
@@ -8,11 +8,10 @@
       head += org.sakaiproject.portal.util.PortalUtils.includeLatestJQuery("Samigo");
       head += "<script type=\"text/javascript\" src=\"/library/webjars/jquery.tablesorter/2.27.7/dist/js/jquery.tablesorter.min.js\"></script>\n";
       head += "<link href=\"/library/webjars/jquery.tablesorter/2.27.7/dist/css/theme.default.min.css\" type=\"text/css\" rel=\"stylesheet\" media=\"all\" />\n";
+      head += "<script type=\"text/javascript\" src=\"/samigo-app/js/jquery.corner.js\"></script>\n";
```

Comment:
do we need jquery.corner when you can use border-radius in pure CSS?

Reply:
I'm using it to turn corners on and off when "attaching" to the top of the frame while scrolling, maybe it could be re-implemented with pure css, but this is what we have in prod :/